src/components/Header/Header.component.jsx

```diff
@@ -0,0 +1,33 @@
+import React from  'react';
+
+import styled from 'styled-components';
+import Menu from './Menu';
+import SearchBar from './SearchBar';
+import RightHand from './RightHand';
+
+
+const HeaderBox = styled.div`
```

Comment:
I'd like to suggest you to keep the styled components in its own file `styled.js` as they can get large for bigger components or have multiple styled components. This way you keep your main component cleaner. Something like

```javascript
// styled.js
export const HeaderBox = styled.div`
  ...
`;

// Header.component.jsx
import { HeaderBox } from './styled';
```